File: BitacoraModels/Roles.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace BitacoraModels
{
    public class Roles
    {
        public int IdRol { get; set; }
        public string Rol { get; set; }
        public string Descripcion { get; set; }
        public string Estatus { get; set; }
        public DateTime FechaRegistro { get; set; }
        public DateTime FechaModificacion { get; set; }
        public List<string> ListPantallas { get; set; }
        public List<int> IdPantallas { get; set; }
    }
}

File: bitacora.client/src/app/models/roles.ts
export class Roles {
  idRol?: Number;
  Rol: String;
  Descripcion: String;
  Estatus?: number;
  FechaRegistro?: Date;
  FechaModificacion?: Date;
  ListPantallas?: Array<String>;
  IdPantallas: Array<Number>;

  constructor(
    idRol: Number | undefined,
    Rol: String,
    Descripcion: String,
    Estatus: number | undefined,
    FechaRegistro: Date | undefined,
    FechaModificacion: Date | undefined,
    ListPantallas: Array<String> | undefined,
    IdPantallas: Array<Number>,
  ) {
    this.idRol = idRol;
    this.Rol = Rol;
    this.Descripcion = Descripcion;
    this.Estatus = Estatus;
    this.FechaRegistro = FechaRegistro;
    this.FechaModificacion = FechaModificacion;
    this.ListPantallas = ListPantallas;
    this.IdPantallas = IdPantallas;
  }
}
